docker-app/qfieldcloud/urls.py

```diff
@@ -29,8 +29,11 @@
 )
 from qfieldcloud.authentication import views as auth_views
 from qfieldcloud.core.views import files_views
+from qfieldcloud.core.views.users_views import *
```

Comment:
Never do this, `*` in imports are forbidden for reasons that google can help with ;)

Reply:
Google helped me ;)

Reply:
Changed!